src/storage/v3/shard.hpp

```diff
@@ -227,36 +225,31 @@ class Shard final {
                               const std::optional<utils::Bound<PropertyValue>> &lower_bound,
                               const std::optional<utils::Bound<PropertyValue>> &upper_bound, View view);
 
-    /// Return approximate number of all vertices in the database.
-    /// Note that this is always an over-estimate and never an under-estimate.
-    int64_t ApproximateVertexCount() const { return static_cast<int64_t>(shard_->vertices_.size()); }
+    /// Return number of all vertices in the database.
+    int64_t VertexCount() const { return static_cast<int64_t>(shard_->vertices_.size()); }
```

Comment:
I think I made a mistake here, it is still approximate, since it does not verify if the vertex is deleted or not, I am right @antaljanosbenjamin ?

Reply:
Yes, that is true. Furthermore be suspicious about errors regarding deleted vertices. I think this change shouldn't cause any issues, because we only removed the vertices from the `SkipList` when we were sure there is no active transaction that should access the vertex, but it is possible that the lack of `SkipList` (concretely that accessors of the `SkipList` will provide access to items that are removed by a future accessor) will cause some issues.